recipes/corrade/all/cmake/conan-corrade-vars.cmake

```diff
@@ -0,0 +1,61 @@
+
+# Here we are reproducing the variables and call performed by the FindCorrade.cmake provided by the library
+
```

Comment:
that's... an interesting way to define variables. I am curious why did you do it this (original) way?

Reply:
The library provides a `FindCorrade.cmake` which populates all these extra variables (with this mechanism, I've just copy-pasted it), and they are used by consumers (mainly by `magnum` which is a library from the same author).

You can check my draft recipe for `magnum` (and some other deps I'll split into their own PRs one by one) here: https://github.com/conan-io/conan-center-index/pull/6292